components/menu/index.css

```diff
@@ -12,8 +12,10 @@ governing permissions and limitations under the License.
 
 .spectrum-Menu {
 	--spectrum-menu-item-background-color-default-rgba: rgba(var(--spectrum-menu-item-background-color-default), var(--spectrum-menu-item-background-opacity));
+	--spectrum-menu-item-background-color-key-focus: var(--spectrum-menu-item-background-color-keyboard-focus);
```

Comment:
It seems a little odd that the token is named with `keyboard-focus` instead of `key-focus` like most other tokens I've seen, so I'm using it to fit with the custom property we're already using for these styles

Reply:
Agreed. It looks like there might just be some inconsistency though, too. In combobox, it's opposite 😆 Although these examples are specifically the `mod` properties, so maybe it's purposely different.

```
--mod-textfield-text-color-keyboard-focus: var(--mod-combobox-font-color-key-focus);
--mod-textfield-border-color-invalid-keyboard-focus: var(--mod-combobox-border-color-invalid-key-focus, var(--spectrum-combobox-border-color-invalid-key-focus));
```